samples/peripheral/802154_phy_test/prj.conf

```diff
@@ -26,7 +26,7 @@ CONFIG_NRFX_TIMER2=y
 CONFIG_NRF_802154_RADIO_DRIVER=y
 
 # Allow sharing the RTC between IEEE 802.15.4 and Zephyr
-CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=3
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
```

Comment:
This is incorrect for platforms other than `CONFIG_SOC_NRF5340_CPUNET`
This is correct only for `CONFIG_SOC_NRF5340_CPUNET`

Reply:
Fixed